src/Assert.php

```diff
@@ -24,9 +24,13 @@
 use Throwable;
 use Traversable;
 
+require_once __DIR__ . '/Mixin.php';
```

Comment:
Psalm does not autoload it, probably should be reported to psalm (I'm just a bit afraid of overwhelming Matthew with bug reports :-D )

Reply:
After some more research, it looks like it does not autoload it because some internal collisions: psalm from `ci` directory brings its own copy of `Assert`, while we also have an `Assert` in our currently developed project, and psalm decides to not autoload it :shrug: 

In a simple example without multiple class declarations it works, and when we have multiple versions - it does not.

Not sure I know composer (and psalm internals?) well enough to know how to avoid it.

Reply:
Ok, after debugging psalm - it looks like at the moment it does not autoload it, by design.

Reply:
Upd: reported https://github.com/vimeo/psalm/issues/3174